cosmwasm/contracts/crosschain-registry/src/state.rs

```diff
@@ -0,0 +1,5 @@
+use cw_storage_plus::Map;
+
+pub const CONTRACT_MAP: Map<&str, String> = Map::new("contract_map");
+pub const CHAIN_CHANNEL_MAP: Map<&str, String> = Map::new("chain_channel_map");
+pub const ASSET_MAP: Map<&str, String> = Map::new("asset_map");
```

Comment:
I find it useful to add comments / docs to states to explain what the keys and values represent.

Oh, and in rust `//` is a comment and `///` is a doc (a comment that rustdoc will understand when generating doc pages)